hermes-console-vue/src/views/topic/schema-panel/SchemaPanel.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+  const props = defineProps<{
+    schema: string;
+  }>();
+
+  const niceJson = JSON.parse(props.schema);
+</script>
+
+<template>
+  <v-expansion-panels>
+    <v-expansion-panel :title="$t('topicView.schema.title')">
+      <v-expansion-panel-text>
+        <pre>
+          <v-code class="raw-schema-snippet">{{ niceJson }}</v-code>
+        </pre>
+      </v-expansion-panel-text>
+    </v-expansion-panel>
+  </v-expansion-panels>
+</template>
+
+<style scoped lang="scss">
+  .raw-schema-snippet {
+    line-height: 1.4;
+    max-height: 500px;
+    overflow-y: scroll;
+    overflow-x: scroll;
```

Comment:
```suggestion
    overflow: scroll;
```
I am not 100% sure, but I guess we can simplify it like this. Correct me, if I'm wrong.
![image](https://github.com/kszapsza/hermes/assets/44726730/879ec5f2-4298-4651-846e-8653684939c9)